.github/workflows/lint-mixins.yml

```diff
@@ -75,8 +75,12 @@ jobs:
 
       - name: Install Mixin dependencies
         working-directory: ./${{ matrix.mixin }}
-        run: jb install
+        run: make build
 
-      - name: Lint Mixin
+      - name: Lint Mixin with Mixtool
         working-directory: ./${{ matrix.mixin }}
         run: mixtool lint mixin.libsonnet
+
+      - name: Check for unexpected changes in generated files
+        working-directory: ./${{ matrix.mixin }}
+        run: make && git diff --exit-code
```

Comment:
nitpick, but maybe include a message for how to fix CI (e.g. `"Run `make` and commit the local diff"`)

Reply:
updated